guides/common/modules/con_managing-dhcp-using-smart-proxies.adoc

```diff
@@ -0,0 +1,21 @@
+[id="Managing_DHCP_Using_Smart_Proxies_{context}"]
+= Managing DHCP Using {SmartProxies}
+
+{Project} can integrate with a DHCP service using your {SmartProxy}.
+The {SmartProxy} has multiple providers, allowing you to integrate with your existing DHCP infrastructure or deploy a new one.
```

Comment:
Should be generic?
```suggestion
A {SmartProxy} has multiple providers, allowing you to integrate with your existing DHCP infrastructure or deploy a new one.
```

Also, would it make sense to mention that one `{SmartProxy}` can only use one provider active at a time?

Reply:
Applied. IMHO only one service is a given due to line 4.